Updated the Collections section of Privacy whitepaper (#2778)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

Collections storage and feature availability depend on your Microsoft Edge Sign-in and Sync settings.

<!-- ------------------------------ -->
#### Signed in with a personal account, and Sync enabled

When you sign in to Microsoft Edge with a personal Microsoft Account (MSA) and enable Sync, your collections, saved items, notes, and images are automatically saved on Microsoft cloud for free. You can access your collections from Microsoft Edge on any device where you're signed in with Sync enabled. You can even view and manage your collections online in any browser by visiting the [Collections portal](https://www.bing.com/saves) at Bing.com. Collections also enhance your experience with a personalized interest feed, if your personalization settings support this feature.

> If you want to disable Collections for a signed-in profile using a personal account with Sync enabled, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.

Author: @apryzgoda - instead of adding sentence per Suggestion:
"When using a Microsoft Account (MSA), collections aren't available when using InPrivate browsing."
added sentence higher above from Jason:
"You cannot access Collections from an InPrivate window while signed in with a personal account."

latest Files Changed view:
https://github.com/MicrosoftDocs/edge-developer/pull/2778/files

closing

> If you want to disable Collections for a signed-in profile using a personal account with Sync enabled, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.

Author: per Jason:

we remove the sync option for Collections when a user is on Prism since we don't use sync

> If you want to disable Collections for a signed-in profile using a personal account with Sync enabled, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.

Author: incorp'd rewrite of the 3 signin sections from @apryzgoda

<!-- ------------------------------ -->
#### Signed in with a work or school account, and Sync enabled

> If you're signed in with a work or school account and are syncing Collections, your collections, saved items, notes, and images are synced across devices. If you want to stop syncing your Collections, you can turn off Sync for Collections from `edge://settings/profiles/sync`.

Author: done locally

<!-- ------------------------------ -->
#### Signed-out, or Sync disabled

> When you don't sign in to Microsoft Edge, or when you sign in with Sync disabled, all collections data is stored locally on the device, organized per Microsoft Edge profile.

Author: done locally

Collections aren't available when using Guest mode.

> When using a Microsoft Account (MSA), collections aren't available when InPrivate browsing.  InPrivate browsing is supported when using a Microsoft Azure Active Directory (AAD) account, or when not signed in.

Author: ```suggestion
When using a Microsoft Account (MSA), collections aren't available when InPrivate browsing.  Collections are available when using InPrivate browsing, if you are using a Microsoft Azure Active Directory (AAD) account, or when you aren't signed in.
```
@yuanjunz - tech review this change, which shifts the emphasis from InPrivate support to Collections support.  Not sure if the existing final sentence is correct; depends on interpretation of "It is avoidable" in the notes:

Orig notes:
Collections aren't available when using Guest mode or InPrivate browsing. => expand below
Collections aren't available when using Guest mode -- for all condition
Collections aren't available when InPrivate browsing -- this is only for Microsoft account case (MSA), It is avoidable for AAD and non sign in.

Author: essentially done; redistributed the points.  latest view:
https://github.com/MicrosoftDocs/edge-developer/pull/2778/files

> When using a Microsoft Account (MSA), collections aren't available when InPrivate browsing.  InPrivate browsing is supported when using a Microsoft Azure Active Directory (AAD) account, or when not signed in.

Author: done; latest Files Changed view:
https://github.com/MicrosoftDocs/edge-developer/pull/2778/files

<!-- Above, should pick one way. Check UI for privacy updates -->

> If you're signed in with a Microsoft account, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, disable Collections for this device as follows: go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.  Collections aren't available when using Guest mode or InPrivate browsing for your Microsoft account.

Author: Add a paragraph about the behavior for when you're signed in with a work or school account.
@yuanjunz

Reviewer: When you sign in with a work or school account, your collections are automatically synced across all signed-in Microsoft Edge devices when sync is turned on. If you want to stop syncing your Collections, you can turn off sync for Collections from `edge://settings/profiles/sync`, then all Collections data is stored locally with your Microsoft Edge profile on the device.

Author: added

> If you're signed in with a **Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, disable Collections for this device as follows: go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.  Collections aren't available when using Guest mode or InPrivate browsing for your **Microsoft account**.

Author: Add a paragraph about the behavior for when you're signed in with a work or school account.
@yuanjunz

> If you're signed in with a **personal Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, go to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  When this setting is turned off, Collections are disabled on this device.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.

Author: @yuanjunz -
Which section of Edge Settings contains the **Show Collections and follow content creators in Microsoft Edge** toggle?
Which version of Edge has the toggle?  I didn't find it in my org-managed Edge Stable or Edge Canary.  I'm not signed in with a **personal Microsoft account**,

Reviewer: This is for personal Microsoft Account (MSA). this feature is called new Collections (Prism collections), since Edge 111, legacy Collections MSA users will be updated (migrated) to new Collections. They have this option
<img width="1053" alt="image" src="https://github.com/MicrosoftDocs/edge-developer/assets/13581525/93de91d0-b926-4fb8-a169-388c9670df24">

Author: updated to:
go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.

> If you're signed in with a **personal Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, go to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  When this setting is turned off, Collections are disabled on this device.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.

Reviewer 2: Regarding: "automatically saved on Microsoft cloud for free". I found the following statement in another part of the Collections section of the whitepaper: "All collections data is stored locally on the device and organized per Microsoft Edge profile. If you have sync turned on for Collections, your collections, notes, and comments are available across all signed-in and syncing versions of Microsoft Edge.". Are these two statements in opposition, or are they both true? Either way, I think this addition to the whitepaper requires us to edit that section as well. With your clarification, I can help edit/re-write.

Author: @yuanjunz - see above comment

Reviewer: To help you understand, below are background
We have 3 types of Collections data.

- 1, For non sign in users, they are using Classic collections without sync. So statement: All collections data is stored locally on the device and organized per Microsoft Edge profile.

- 2, For AAD sign in users, they are using Classic collections (legacy mode). 
- - if sync is not turned on. Their collections data is stored locally. 
- - if sync is turned on. Local stored collections data will sync to Edge sync cloud. 

- 3 For MSA sign in users, 
- - if sync is not turned on. Their collections data is stored locally. 
- - if sync is turned on. Their Collections data is updated to New Collections (migrated to Prism collections). And synced to Microsoft Bing Saves cloud.

So, "All collections data is stored locally on the device and organized per Microsoft Edge profile." => means all Classic Collections (legacy Collections) data are stored locally, except new Collections (Prism collections). Above # 1

"If you have sync turned on for Collections, your collections, notes, and comments are available across all signed-in and syncing versions of Microsoft Edge." => means for AAD signed in and sync turned on users, Collection data is synced from local store to Edge sync cloud. Above # 2.

"If you're signed in with a Microsoft account, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free" => Means for MSA signed in and sync turned on users. new Collections (Prism Collections) data is saved to Microsoft Bing Saves cloud. Above # 3

Author: @apryzgoda - I added the above info, and added hidden comments to identify Sentence A, B, & C, displayed as quoted sentences near bottom of section (like in the above comment).  We want to not quote those entire sentences A, B, & C.  Ideas for reorganizing the Collections section?

Author: Incorp'd rewrite from Alyssa

> If you're signed in with a **personal Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, go to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  When this setting is turned off, Collections are disabled on this device.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.

Reviewer 2: ```suggestion
If you're signed in with a **personal Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, disable Collections for this devices by going to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.
```

Author: done locally as:
If you're signed in with a **personal Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, disable Collections for this device by going to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turning off the toggle for **Show Collections and follow content creators in Microsoft Edge**.<!--todo: no such setting-->.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.

> If you're signed in with a **personal Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, go to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  When this setting is turned off, Collections are disabled on this device.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.

Reviewer 2: I recommend removing "personal" here, and then adding a section about the behavior for when your'e signed in with a work or school account. 

```suggestion
If you're signed in with a **Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, go to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  When this setting is turned off, Collections are disabled on this device.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.
```

Author: @yuanjunz - see the above Comment.  I don't know if you're able to push a commit to this PR that I created (maybe I should have created a forked-repo PR and then Invited you to contribute to my forked repo).  

You can paste draft text here, or to go fresh Files Changed tab of this PR and then enter a Comment that's a Suggested Change (click [+/-] icon in comment's toolbar).

The draft text could be a paragraph, about the behavior for when you're signed in with a work Microsoft account or school Microsoft account instead of personal Microsoft account.

Another article has wording:
Use a personal Microsoft account (MSA) (not a work Microsoft account (MSA) or a school Microsoft account (MSA)) to register
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/d1d0843e-e715-474c-b9c8-13bc77af232a)

Reviewer: Agree to removing "personal" here, use **Microsoft account**

Author: removed 'personal' 2x.
Didn't do:
* Add a section about the behavior for when you're signed in with a work or school account.

Author: "Microsoft account" occurs 23x in https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/, not bold.

Please contact your enterprise administrator to understand your company's policies for such data collection.

<!-- ====================================================================== -->
## Entity extraction

> Every 24 hours, Microsoft Edge downloads a list of supported sites that have special entity extraction templates.  The templates are specific to each website.  When you create a new item in your collection, Microsoft Edge verifies that the site you are collecting from is in the list of supported sites.  If the site is in the list, Microsoft Edge pings the entity extraction service for the template of the specific site.  No user identifiers are associated with the request to the service.  The template attempts to identify the name, price, ratings, primary image, and other data about the item being collected.  If the site you are collecting from is not on the supported list site, Microsoft Edge does not download a template.  Templates allow collection items to be created locally on the device.  No data about the collection items are sent to the service when creating the collection.

Author: @yuanjunz - need E E team to review.  "collection" occurs 8x in this paragraph.

Paragraph needs to be split up for readability.

Reviewer: Simplify below:
Microsoft Edge uses entity extraction templates specific to a list of supported websites to identify the name, price, ratings, primary image, and other data about the item being collected locally.  When creating the collection, no user identifiers or other data are sent to Microsoft services.
You can delete templates stored on the device by going to `edge://settings/clearBrowserData`, selecting the desired time range and type of data, and then selecting the **Clear now** button.

Author: done.
entered [fresh Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2778/files#r1300819539)

Please contact your enterprise administrator to understand your company's policies for such data collection.

<!-- ====================================================================== -->
## Entity extraction

Every 24 hours, Microsoft Edge downloads a list of supported sites that have special entity extraction templates.
The templates are specific to each website.
When you create a new item in your collection, Microsoft Edge verifies that the site you are collecting from is in the list of supported sites.
If the site is in the list, Microsoft Edge pings the entity extraction service for the template of the specific site.
No user identifiers are associated with the request to the service.
The template attempts to identify the name, price, ratings, primary image, and other data about the item being collected.
If the site you are collecting from is not on the supported list site, Microsoft Edge doesn't download a template.
Templates allow collection items to be created locally on the device.
No data about the collection items are sent to the service when creating the collection.

> <!-- todo: check "collect" above (8x), broaden focus beyond collections? -->

Author: @yuanjunz 
todo: check "collect" above (8x), broaden focus beyond collections?

todo: break into multiple paragraphs for readability: 
1. Group and arrange sentences in logical order.
1. Add blank lines between paragraphs.  Aim for 1-3 short sentences per paragraph.
1. Concat the sentences in each paragraph.
1. If needed, add some h4 "#### Foo" section headings.

Reviewer: ## Entity extraction

Microsoft Edge uses entity extraction templates that are specific to a list of supported websites to identify the name, price, ratings, primary image, and other data about the item being collected locally. When creating the collection, no user identifiers or other data are sent to Microsoft services.

If you want to delete such templates stored on the device, go to `edge://settings/clearBrowserData`, select the desired time range and type of data, and then click the **Clear now** button.

Author: deleted paragraph
